frontend/nuxt.config.js

```diff
@@ -23,14 +26,42 @@ export default {
 
     // Global page headers: https://go.nuxtjs.dev/config-head
     head: {
-        title: 'Auctions UI',
+        title: 'Unified Auctions UI',
```

Comment:
Is it `Unified Auctions` or `Unified Auctions UI`? I would skip the UI part as it's not completely true, also doesn't correspond to our subdomain.

Note: I would also move this into variable on top of the file, like you did for the `SITE_DESCRIPTION`

Reply:
Sounds good!